src/components/LoadingBar.tsx

```diff
@@ -57,7 +57,7 @@ function LoadingBar({shouldShow}: LoadingBarProps) {
     }));
 
     return (
-        <View style={[styles.progressBarWrapper]}>
+        <View style={[styles.progressBarWrapper, {opacity: shouldShow ? 1 : 0}]}>
```

Comment:
I don't know that I love the opacity trick, because in theory, it's still a 1px tall area that could potentially block your interactions with the rest of the visible UI. I know it's being nitpicky, but I don't think it's the most proper way to handle this. I would still toggle height from 0 to 2, so that when the height is 0 it has essentially disappeared from being an element you can interact with.

Reply:
Sounds good, thanks for hearing me out!